src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ElementCompletionContext.cs

```diff
@@ -6,12 +6,14 @@
 using System;
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.Language.Syntax;
 
 namespace Microsoft.VisualStudio.Editor.Razor
 {
     public sealed class ElementCompletionContext
     {
-        public ElementCompletionContext(
+        internal ElementCompletionContext(
```

Comment:
`internal` because `SyntaxNode` is internal and otherwise we get compile problems.

Reply:
This will break WebTools 😢. They use this type. We could add another constructor though